### PR TITLE
Hide the java link if the container is not ready

### PR DESCRIPTION
**Pull request by gashcrumb**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1270491

@jwforres FYI, thanks!

### CODE REVIEW

File: assets/app/scripts/extensions/javaLink.js [hunk 1]
```diff
@@ -52,6 +52,13 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
       if (!pod || !pod.status || pod.status.phase !== 'Running') {
         return;
       }
+      var containerStatuses = pod.status.containerStatuses;
+      var containerStatus = _.find(containerStatuses, function(status) {
+        return status.name === container.name;
+      });
+      if (!containerStatus || !containerStatus.ready) {
+        return;
+      }
```
Comment (spadgett) [inline assets/app/scripts/extensions/javaLink.js]: Maybe just...

```
if (!_.get(pod, ['status', 'containerStatuses', name, 'ready'])) {
  return;
}
```

Comment (spadgett) [inline assets/app/scripts/extensions/javaLink.js]: (not tested)

Comment (gashcrumb) [inline assets/app/scripts/extensions/javaLink.js]: Will _.get handle the fact that containerStatuses is a regular indexed array?

Comment (spadgett) [inline assets/app/scripts/extensions/javaLink.js]: No, sorry, you're right. Was thinking it was an object.

**Comment by spadgett** [comment]
[merge]

File: assets/app/scripts/extensions/javaLink.js (hunk 1 above)
Comment (gashcrumb) [inline assets/app/scripts/extensions/javaLink.js]: no worries, would be nicer if it was a map :-)

**Pull request merged**